src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -128,7 +128,7 @@
      * FIXME: remove eventually if not used anymore
      * The list of {@link FocusAllocationListener}.
      */
-    private FocusAllocationListener focusAllocListener;
+    private List<FocusAllocationListener> focusAllocListener = new ArrayList<FocusAllocationListener>();
```

Comment:
Thats fair...I didn't change it since I thought the refactor might be a pain...I'll go ahead and do it though.